parm/input.nml.FV3

```diff
@@ -207,28 +207,6 @@
 /
 
 &namsfc
-    fabsl = 99999
```

Comment:
Is there any reason to keep `&namsfc` in, even though its contents are being removed? (e.g., does the WM expect to see it whether or not it has contents?)

Reply:
@gspetro-NOAA I ran a test with the full removal of `&namsfc` and there were no EOF error while reading the `input.nml` file in the fundamental WE2E tests, so I will go ahead and fully remove it.